### DISCUSSION TITLE
GitHub Licensing and Wiki Contributors

### DISCUSSION THREAD

**Discussion by Jenkj3** [community/community#21930] [Other Feature Feedback, Questions, & Ideas]
<p>Hello GitHub Pages Community!</p>
<p>The organization I work for is considering a GitHub Pages site for a process “Playbook” Wiki.</p>
<p>The goal is to crowdsource best practices and employee experiences from hundreds of employees across the enterprise.</p>
<p>Some questions have popped up about whether our Enterprise license cost will be affected by promoting the Wiki to hundreds (potentially thousands) of employees asking them to contribute and edit the Pages site.</p>
<p>A moderator/facilitator will monitor all contributions before a commit is performed.</p>
<p>Can anyone confirm that indivdual Wiki contributors do not affect the number of user licences needed as long as an existing user with a license acts as the moderator?</p>
<p>Any insight will be greatly appreciated.</p>

**Comment by mpboom** [comment, marked as answer]
<p>Well, I wouldn’t think so, but since you have an Enterprise plan you might have additional terms that apply? You could maybe reach out to GitHub and let they confirm it? Since we on the forum don’t know what contract you signed with GH.</p>

**Comment by nadiajoyce** [comment]
<p>Hi <a class="mention" href="/jenkj3">@jenkj3</a> </p>
<p>For questions about a specific GitHub account, you’ll need to reach out to private support. It sounds like you are working on an Enterprise plan? If so, you’ll want to touch base with your administrator and/or seek support from GitHub Enterprise Support: <a href="https://enterprise.github.com/support" rel="nofollow noopener">https://enterprise.github.com/support</a></p>
<p>Thanks!</p>

**Comment by Jenkj3** [comment]
<p>Thanks Mark!</p>
<p>I’m new to GitHub and your response on how to move forward on the issue is very helpful.</p>
<p>Thanks for your comments.</p>

**Comment by Jenkj3** [comment]
<p>Hi <a class="mention" href="/nadiajoyce">@nadiajoyce</a></p>
<p>Thanks for responding to my questions.  I’ve located our organizations licensing manager responsible for GitHub and will work through her to contact Enterprise Support as you suggested.  Thanks for supplying the link and your guidance on this issue.</p>

**Comment by nadiajoyce** [comment]
<p><a class="mention" href="/jenkj3">@jenkj3</a> You bet! Any time :slight_smile:</p>
